x-pack/plugins/spaces/public/management/spaces_management_app.tsx

```diff
@@ -63,16 +62,7 @@ export const spacesManagementApp = Object.freeze({
 
         chrome.docTitle.change(title);
 
-        const onCloud = Boolean(cloud?.isCloudEnabled);
-        const isSolutionNavEnabled$ =
-          // Only available on Cloud and if the Launch Darkly flag is turned on
-          onCloud && cloudExperiments
-            ? from(
-                cloudExperiments
-                  .getVariation(SOLUTION_NAV_FEATURE_FLAG_NAME, false)
-                  .catch(() => false)
-              ).pipe(shareReplay(1))
-            : of(false);
+        const isSolutionNavEnabled$ = isSolutionNavEnabled(cloud, cloudExperiments);
```

Comment:
> If cloudExperiments could expose getVariation from both setup and start
agree that would be much easier

About the code, I'm okay to make the change, but I suppose it would be better if we agree whether we want to convert experiment to pure promise or keep it as is. 

@sebelga I'm a bit new to our experimentation setup for kibana, since you've setup an experiment initially, could you please clarify if that would be acceptable to convert it to promise? Will it have any impact on experiment traffic allocation?

Reply:
All good to use Promise instead of Observable. I like to use Observable when I have several state in Observable format (e.g. `activeSpace$`). It makes things easier for me to handle, but that's kind of personal preference 😊